dynaconf/default_settings.py

```diff
@@ -246,6 +246,8 @@ def reload(load_dotenv=None, *args, **kwargs):
     "APPLY_DEFAULT_ON_NONE_FOR_DYNACONF", None
 )
 
+VALIDATE_ON_UPDATE_FOR_DYNACONF = get("VALIDATE_ON_UPDATE", False)
```

Comment:
I think the variable must be with suffix here

get function could be smart to add missing suffix but I think it needs now.

```suggestion
VALIDATE_ON_UPDATE_FOR_DYNACONF = get("VALIDATE_ON_UPDATE_FOR_DYNACONF", False)
```